submit_for_testing.py

```diff
@@ -120,15 +120,16 @@ def _submit_to_squad(lava_job, lava_url_base, qa_server_api, qa_server_base, qa_
             ).netloc,  # qa-reports backends are named as lava instances
         }
         logger.info("Submit to: %s" % qa_server_api)
-        results = requests.post(qa_server_api, data=data, headers=headers, timeout=31)
-        if results.status_code < 300:
-            logger.info(
-                "%s/testjob/%s %s"
-                % (qa_server_base, results.text, get_job_name(lava_job))
+        try:
+            results = requests.post(
+                qa_server_api, data=data, headers=headers, timeout=31
             )
-        else:
-            logger.info(results.status_code)
-            logger.info(results.text)
+        except requests.exceptions.HTTPError as e:
+            logger.error("QA Reports error, with error code: " % results.status_code)
+            raise SystemExit(e)
```

Comment:
I think its best to just call `raise` here to re-raise the HTTPError exception.

```
raise
```

instead of

```
raise SystemExit(e)
```

Reply:
I use 'SystemExit(e)' so we get a full stack trace why we did exit.

Reply:
I think it is a better practice to log why we exited (since we will know when we raise the exception) rather than raising an exception. 

I guess what I'm getting at is, What extra info will the exception provide that the logged message will not? Additionally, what do we do in other areas of the code that are similar?

I think I'm still OK with doing this, however!